portfolio_ahardman_stylesheet.css

```diff
@@ -0,0 +1,339 @@
+/*   default settings   */
+:root{
+	--cyan_light: rgb(0, 204, 191);
+	--cyan_dark: rgb(0, 164, 153);
+}
+*{
+	box-sizing: border-box;
+	font-family: 'Segoe UI', Helvetica, Verdana; 
+}
+body{                 	/* removes default padding and margin around the ouside of the entire webpage */
+	font-size: 62.5%;
+	margin: 0;
+	padding: 0;
+	background-image: linear-gradient(to right,rgb(110,110,110),rgb(180,180,180));
+}
+@-ms-viewport{
+	width: device-width;
+}
+a{
+	text-decoration: none;
+	color:rgb(0,0,0);
+}
+input:focus{
+	border: none;
+	outline: none;
+}
+svg.logo{
+	z-index: -1;
```

Comment:
negative z-index is generally a bad idea. this actually does nothing in this case